azurerm_monitor_action_group/output.tf

```diff
@@ -1,3 +1,3 @@
 output "name" {
-  value = "${var.azurerm_monitor_action_group_name}"
+  value = "${var.azurerm_monitor_action_group_name_suffix}"
```

Comment:
This is wrong:
* The output name doesn't respect conventions (should be azurerm_monitor_action_group_name)
* The value is also wrong. The be the local one, not the suffix
* Description missing